sdk/keyvault/keyvault-keys/src/keysModels.ts

```diff
@@ -285,10 +287,16 @@ export interface KeyReleasePolicy {
    */
   contentType?: string;
 
-  /** Blob encoding the policy rules under which the key can be released. */
+  /**
+   * The policy rules under which the key can be released. Encoded based on the {@link KeyReleasePolicy.contentType}.
+   *
+   * For more information regarding the release policy grammar for Azure Key Vault, please refer to:
```

Comment:
"grammar" works, but I wonder if "schema" would be better. It's more oft used. Either is fine, though.